src/index.js

```diff
@@ -91,6 +107,37 @@ class GossipSub extends BasicPubsub {
     this.heartbeat = new Heartbeat(this)
   }
 
+  /**
+   * Given a topic, returns up to count peers subscribed to that topic
```

Comment:
We can probably get this its own file? What do you think?

Reply:
:+1:  can move to its own file. In gossipsub v1.1 i think this is going to have to change (return an Array of peers sorted by score instead of a Set)